include/roboteam_ai/control/positionControl/PositionControl.h

```diff
@@ -0,0 +1,71 @@
+//
+// Created by ratoone on 18-11-19.
+//
+
+#ifndef RTT_POSITIONCONTROL_H
+#define RTT_POSITIONCONTROL_H
+
+
+#include "control/RobotCommand.h"
+#include "control/positionControl/pathPlanning/NumTreesPlanning.h"
+#include "control/positionControl/pathTracking/BasicPathTracking.h"
+#include "CollisionDetector.h"
+#include <world_new/Robot.hpp>
+#include <control/positionControl/pathTracking/NumTreesTracking.h>
+
+
+namespace rtt::ai::control {
+
+/**
+ * The main position control class. Use this for your robot position control
+ * requirements.
+ */
+class PositionControl {
+private:
+
+    const double MAX_DEVIATION = 0.3;
```

Comment:
This si quite a lot of deviation. 0.3 meters is a lot.

Reply:
constexpr static

Reply:
0.3m was the deviation in the old algorithm. I can decrease it to 0.2m

Reply:
No keep it like that then but move it to numtrees files I would say. It does not make a lot of sense for it to live here.

Reply:
It is used though in the condition if the path should be recalculated - which does not depend on the planning method. Maybe I can rename it to show that